environment.yml

```
@@ -41,7 +41,7 @@ dependencies:
   - nc-time-axis
   - netCDF4
   - numba
-  - numpy !=1.24.3,<2.0  # severe masking bug
+  - numpy !=1.24.3
```

```suggestion
  - numpy !=1.24.3  # severe masking bug
```

Leave comment like for OSX environment?